api/types/network/endpoint.go

```diff
@@ -24,6 +28,7 @@ type EndpointSettings struct {
 	IPv6Gateway         string
 	GlobalIPv6Address   string
 	GlobalIPv6PrefixLen int
+	MacAddress          string
```

Comment:
No need to move that field, it's still what should be used by API clients to configure a container's MAC address.